### PR TITLE
fix: egctl stats invalid format

### PR DESCRIPTION
**Pull request by Xunzhuo**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

+ Fix json format output is invalid
+ Clean-up unused prom output type
+ Add alias and short for stats commands

### CODE REVIEW

File: internal/cmd/egctl/envoy_stats.go [hunk 1]
```diff
@@ -124,7 +129,7 @@ func newEnvoyStatsCmd() *cobra.Command {
 			return nil
 		},
 	}
-	statsConfigCmd.PersistentFlags().StringVarP(&outputFormat, "output", "o", summaryOutput, "Output format: one of json|yaml|prom")
```
Comment (zirain) [inline internal/cmd/egctl/envoy_stats.go:127]: previous is intend, should change the description.

Comment (Xunzhuo) [reply internal/cmd/egctl/envoy_stats.go:127]: We support three types json/yaml/prom, the default value should be set with `prometheusOutput`, and remove the unused `summaryOutput`. What should we update here? I do not get it.

Comment (zirain) [reply internal/cmd/egctl/envoy_stats.go:127]: we don't need support json/yaml on stats.

File: internal/cmd/egctl/stats.go [hunk 2]
```diff
@@ -11,8 +11,10 @@ import (
 
 func statsCommand() *cobra.Command {
 	c := &cobra.Command{
-		Use:  "stats",
-		Long: "Retrieve stats from envoy proxy.",
+		Use:     "stats",
+		Long:    "Retrieve Prometheus Stats from envoy proxy.",
+		Short:   "Retrieve stats from envoy proxy.",
+		Aliases: []string{"proxy"},
```
Comment (zirain) [inline internal/cmd/egctl/stats.go:17]: this line seems meaningless.

File: internal/cmd/egctl/envoy_stats.go (hunk 1 above)
Comment (Xunzhuo) [reply internal/cmd/egctl/envoy_stats.go:127]: Not sure about that, so why adding json/yaml before cc @tanujd11 ?

File: internal/cmd/egctl/stats.go (hunk 2 above)
Comment (Xunzhuo) [reply internal/cmd/egctl/stats.go]: Done

**Review by zirain — DISMISSED** [review]

File: internal/cmd/egctl/stats.go [hunk 3]
```diff
@@ -11,8 +11,9 @@ import (
 
 func statsCommand() *cobra.Command {
 	c := &cobra.Command{
-		Use:  "stats",
-		Long: "Retrieve stats from envoy proxy.",
+		Use:   "stats",
+		Long:  "Retrieve Prometheus Stats from envoy proxy.",
```
[the remark this answers is not in the document]
Comment (Xunzhuo) [reply internal/cmd/egctl/stats.go:15]: No matter the type is Prometheus/Json/Yaml, all of them are just Prometheus data format when dumping.

[the remark this answers is not in the document]
Comment (zhaohuabing) [reply internal/cmd/egctl/stats.go:15]: Sorry, I didn't get it. Not an expert of telemetry, so please correct me if I'm wrong. 

I mean that the stats data is generated by Envoy, and then the data can be consumed by Prometheus, Skywalking, or any Open Telemetry compatible backends. So why emphasize Prometheus here?

[the remark this answers is not in the document]
Comment (Xunzhuo) [reply internal/cmd/egctl/stats.go]: Thanks, done.

**Review by arkodg — APPROVED** [review]

**Pull request merged**